src/commands/scan-iac.yml

```diff
@@ -0,0 +1,14 @@
+description: Scan your applications IaC files for issues
+parameters:
+  path:
+    description: Path to a specific file or directory to test
+    type: string
+    default: ""
+steps:
```

Comment:
Comparing this to: https://github.com/snyk/snyk-orb/blob/develop/src/commands/scan.yml

It seems this command is missing a few steps. Namely the install step (and related parameters).